src/main/java/de/zalando/ep/zalenium/container/ContainerClient.java

```diff
@@ -22,11 +22,13 @@
 
     ContainerCreationStatus createContainer(String zaleniumContainerName, String image, Map<String, String> envVars, String nodePort);
 
+    ContainerCreationStatus createContainer(String zaleniumContainerName, String image, String cpuLimit, String memoryLimit, Map<String, String> envVars, String nodePort);
```

Comment:
I don't think we should add the cpu limits this way. I think it might make more sense to import the system variable directly into the docker implementation, or extend the kubernetes implementation to support docker.

https://github.com/zalando/zalenium/blob/master/src/main/java/de/zalando/ep/zalenium/container/kubernetes/KubernetesContainerClient.java#L495

Specifying `100000000` as one cpu seems crazy, maybe we should use the kubernetes definition? And find a way to convert the value:

https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-cpu

Reply:
I agree with @pearj, no need change this methods, a more simple approach is to get the env vars in the `DockerContainerClient.java` class.